README.md

```diff
@@ -36,10 +36,39 @@ Interested in learning more about the basics of virtual environments in Python
 [article](http://www.cs.allegheny.edu/sites/gkapfham/programming/research/idea/2017/07/14/Virtual-Environments/)
 to further develop your understanding of this topic.
 
+GatorGrouper requires users to use Python 3. You can type `python --version` to check
+the current Python version on your device. If you do not have the right Python version,
+or you can not download or upgrade Python on your devices, you can download the
+[Pyenv](https://github.com/pyenv/pyenv) tool to set up a virtual environment for
+the newest Python version.
+
+To install Pyenv, you can use [Pyenv Installer](https://github.com/pyenv/pyenv-installer)
+by typing the command in terminal:
+
+```shell
+curl https://pyenv.run | bash
+```
+
+You will successfully install Pyenv. Please make sure that you have the
+following lines in your `~/.bashrc` file.
+
+```shell
+export PATH="~/.pyenv/bin:$PATH"
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+```
+
+Once Pyenv is installed, you can install the latest version of Python
+for your Pyenv after restart your terminal window by typing `pyenv install 3.6.7`
+and `pyenv global 3.6.7` to install.
+
+After you finish this process, you should be able
+to type `python --version` and get `Python 3.6.7` in your terminal.
+
```

Comment:
Maybe 3.6.8 and 3.7.2?

Reply:
This recommendation has resulted in changes found in [this](https://github.com/Lancasterwu/gatorgrouper/pull/2) pull request.